docs/astro.config.mjs

```diff
@@ -17,6 +17,10 @@ export default defineConfig({
         da: {
           label: "Dansk",
         },
+        es: {
+          label: "Spanish",
```

Comment:
```suggestion
          label: "Español",
```

Reply:
I think it must be Spanish, because the default language is English

Reply:
I think it is a matter of opinion - If the user does not speak English, it would be more useful for them to search for the Spanish word for "Spanish". 

The danish translation is also listed as the danish word "Dansk"